### PR TITLE
Write the Docs: add values for required template variables

### PR DESCRIPTION
**Pull request by eirinikos**
> ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> For 1.7 Features: set Milestone to `1.7` and Base Branch to `release-1.7`
> ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
>
> NOTE: Please check the “Allow edits from maintainers” box below to allow 
> reviewers fix problems on your patch and speed up the review process.
> Please delete this note before submitting the pull request.

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes.github.io/4045)

### CODE REVIEW

File: docs/tasks/access-application-cluster/web-ui-dashboard.md [hunk 1]
```diff
@@ -174,3 +184,5 @@ Pod lists and detail pages link to logs viewer that is built into Dashboard. The
 
 For more information, see the
 [Kubernetes Dashboard project page](https://github.com/kubernetes/dashboard).
+
+{% include tempates/task.md %}
```
Comment (chenopis) [inline docs/tasks/access-application-cluster/web-ui-dashboard.md]: typo: "tempates" -> "templates"

`{% include templates/task.md %}`

File: docs/tasks/access-application-cluster/web-ui-dashboard.md [hunk 2]
```diff
@@ -11,15 +11,25 @@ redirect_from:
...
+Dashboard also provides information on the state of Kubernetes resources in your cluster, and on any errors that may have occurred. This page shows you how to deploy and access the Dashboard UI to then deploy, configure, and manage your containerized applications.
 
 ![Kubernetes Dashboard UI](/images/docs/ui-dashboard.png)
+{% endcapture %}
 
 * TOC
 {:toc}
 
+{% capture prerequisites %}
+*TBA*
```
Comment (chenopis) [inline docs/tasks/access-application-cluster/web-ui-dashboard.md]: Is this PR still a work in progress?

File: docs/tasks/access-application-cluster/web-ui-dashboard.md [hunk 3]
```diff
@@ -11,15 +11,25 @@ redirect_from:
...
+{% endcapture %}
+
+{% capture steps %}
+*TBA*
```
Comment (chenopis) [inline docs/tasks/access-application-cluster/web-ui-dashboard.md]: All the content starting with *Deploying the Dashboard UI* should probably be in this Steps section.

**Comment by steveperry-53** [comment]
@chenopis, Can we close this due to lack of activity?

**Comment by chenopis** [comment]
Yeah. @eirinikos You can reopen this or create a new PR later, if you want.

**Pull request closed without merging**